### ISSUE TITLE
Error: Package path ./dist/pocketbase.cjs is not exported from package

### ISSUE THREAD

**Issue by drank40** [tobicrain/pocketbase-react#12]
When I try to compile a react project using this library I get : 
```
Compiled with problems:

ERROR in ./node_modules/pocketbase-react/dist/esm/index.js 4:0-56

Module not found: Error: Package path ./dist/pocketbase.cjs is not exported from package /home/USER/...../site/node_modules/pocketbase-react/node_modules/pocketbase (see exports field in /home/USER/...../site/node_modules/pocketbase-react/node_modules/pocketbase/package.json)
```

Im using React 18.2.0 and Pocketbase-sdk 0.7.1, installed with yarn

**Comment by tobicrain** [comment]
Hey, I'm working on this, the current solution is to import from 'pocketbase-react/dist/cjs';
I also had an issue trying to save data in Redux using React. I'm now pushing an update that toggles between LocalStorage (react) and AsyncStorage (react-native).

**Comment by tobicrain** [comment]
Hey, I just released version 0.1.2, everything should work fine now. Feel free to write me if everything fits or if something doesn't work

**Comment by tobicrain** [comment]
@drank40 Hey, please update to version 0.1.5 to use with React and React-Native without problems
